<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>835</width>
    <height>642</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Helvetica</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Растеризация фигур на плоскости</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="leftMargin">
     <number>24</number>
    </property>
    <property name="topMargin">
     <number>24</number>
    </property>
    <property name="rightMargin">
     <number>24</number>
    </property>
    <property name="bottomMargin">
     <number>24</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="upscalePushButton">
          <property name="text">
           <string>Увеличить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="downscalePushButton">
          <property name="text">
           <string>Уменьшить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="gridPushButton">
          <property name="text">
           <string>Сетка вкл</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="diffPushButton">
          <property name="text">
           <string>Показать разницу</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="countDrawPixelsPushButton">
          <property name="text">
           <string>Кол-во закрашенных</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="texturePushButton">
          <property name="text">
           <string>Добавить текстуру</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clearPushButton">
          <property name="text">
           <string>Очистить</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="backPushButton">
            <property name="text">
             <string>Назад</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="currentPolygonLabel">
            <property name="text">
             <string>Текущий полигон: 1</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="forwardPushButton">
            <property name="text">
             <string>Вперед</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="drawPushButton">
          <property name="text">
           <string>Нарисовать отрезки</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Отрезки</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="testLinesPushButton">
              <property name="text">
               <string>Начать испытание 1 (0°)</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Сплошные области</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="testAreasPushButton">
              <property name="text">
               <string>Начать испытание 1 (1)</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="pictureALabel">
        <property name="text">
         <string>Рисунок А</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="pictureBLabel">
        <property name="text">
         <string>Рисунок Б</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="bottomMargin">
       <number>8</number>
      </property>
      <item>
       <widget class="QGraphicsView" name="leftGraphicsView"/>
      </item>
      <item>
       <widget class="QGraphicsView" name="rightGraphicsView"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="countPixelsInCellLabel">
        <property name="text">
         <string>Количество пикселей в ячейке: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="countDrawPixelsALabel">
        <property name="text">
         <string>Количество закрашенных пикселей &quot;А&quot;:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="countDrawPixelsBLabel">
        <property name="text">
         <string>Количество закрашенных пикселей &quot;Б&quot;:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="timeSpentALabel">
        <property name="text">
         <string>Затраченное время &quot;А&quot; (t):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="timeSpentBLabel">
        <property name="text">
         <string>Затраченное время &quot;Б&quot; (t):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="quanDiffLabel">
        <property name="text">
         <string>Количественная разность (ΔI):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="relativeQuanDiffLabel">
        <property name="text">
         <string>Относительная количественная разность (µ):</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
